.travis.sh

```diff
@@ -60,19 +60,52 @@ tests_sequence() {
     )
 }
 
+tests_sequence_ios_sim() {
+    title "testing from workspace directory"
+    ( \
+        cd test-ws \
+        && cargo clean \
+        && $CARGO_DINGHY   -d $1 -p auto-ios-aarch64-sim test pass \
+        && ! $CARGO_DINGHY -d $1 -p auto-ios-aarch64-sim test fails \
+        && ! $CARGO_DINGHY -d $1 -p auto-ios-aarch64-sim test \
+    )
+ 
+    title "testing from project directory"
+    ( \
+        cd test-ws/test-app \
+        && cargo clean \
+        && $CARGO_DINGHY   -d $1 -p auto-ios-aarch64-sim test pass \
+        && ! $CARGO_DINGHY -d $1 -p auto-ios-aarch64-sim test fails \
+        && ! $CARGO_DINGHY -d $1 -p auto-ios-aarch64-sim test \
+    )
+ 
+    title "test from workspace directory with project filter"
+    ( \
+        cd test-ws \
+        && cargo clean \
+        && $CARGO_DINGHY   -d $1 -p auto-ios-aarch64-sim test -p test-app pass \
+        && ! $CARGO_DINGHY -d $1 -p auto-ios-aarch64-sim test -p test-app fails \
+        && ! $CARGO_DINGHY -d $1 -p auto-ios-aarch64-sim test -p test-app \
+    )
+}
+
 
 if [ `uname` = Darwin ]
 then
      title "••••• Darwin: ios simulator tests •••••"
      title "boot a simulator"
-     rustup target add x86_64-apple-ios;
+     rustup target add x86_64-apple-ios aarch64-apple-ios-sim;
      RUNTIME_ID=$(xcrun simctl list runtimes | grep iOS | cut -d ' ' -f 7 | tail -1)
-     export SIM_ID=$(xcrun simctl create My-iphone7 com.apple.CoreSimulator.SimDeviceType.iPhone-8 $RUNTIME_ID)
+     export SIM_ID=$(xcrun simctl create My-iphone-se com.apple.CoreSimulator.SimDeviceType.iPhone-SE-3rd-generation $RUNTIME_ID)
      xcrun simctl boot $SIM_ID
-     tests_sequence $SIM_ID
+     if [ "$(uname -m)" = "arm64" ]; then
```

Comment:
I suggest you put `rustup target add aarch64-apple-ios-sim` here and remove it above. This way we only install the relevant target.